srcpkgs/bluez/patches/0003-bluetooth-group-can-use-bluez.patch

```diff
@@ -1,13 +0,0 @@
---- src/bluetooth.conf
-+++ src/bluetooth.conf
-@@ -28,6 +28,10 @@
-     <allow send_destination="org.bluez"/>
-   </policy>
- 
-+  <policy group="bluetooth">
-+    <allow send_destination="org.bluez"/>
-+  </policy>
```

Comment:
I think this patch shouldn't be dropped but updated against new upstream version. Otherwise bluez would only be usable with elogind.

Reply:
Ah, I updated that patch, but I forgot to include it here.